resources/test/fixtures/pyupgrade/UP026.py

```diff
@@ -0,0 +1,48 @@
+# These should be changed
+if True:
+    import mock
+
+if True:
+    import mock, sys
+
+# This goes to from unitest import mock
+import mock.mock
+
+# Mock should go on a new line as `from unittest import mock`
+import contextlib, mock, sys
+
+# Mock should go on a new line as `from unittest import mock`
+import mock, sys
+x = "This code should be preserved one line below the mock"
+
+# Mock should go on a new line as `from unittest import mock`
+from mock import mock
+
+# Should keep trailing comma
+from mock import (
+    mock,
+    a,
+    b,
+    c,
+)
+
+# Should not get a trailing comma
```

Comment:
(It's also ok, in my opinion, for fixes like this to make some assumptions (e.g., inject a trailing comma) if it greatly simplifies the code.)

Reply:
Could this lead to issues in further run throughs? For example if the trailing comma was removed, then the next time the list might be folded into one line. This isnt a HUGE deal, but could be a little frustrating.

Reply:
Of course its your project and your decisions. Let me know how you want it handled.